py/selenium/webdriver/common/selenium_manager.py

```diff
@@ -0,0 +1,89 @@
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import logging
+import re
+import subprocess
+import sys
+from pathlib import Path
+from typing import Tuple
+
+from selenium.common.exceptions import WebDriverException
+
+logger = logging.getLogger(__name__)
+
+
+class SeleniumManager:
+    """
+    Wrapper for getting information from the Selenium Manager binaries.
+    """
+
+    @staticmethod
+    def get_binary() -> Path:
+        """
+        Determines the path of the correct Selenium Manager binary.
+        :Returns: The Selenium Manager executable location
+        """
+        directory = sys.platform
+        if directory == "linux":
+            directory = "linux"
```

Comment:
minor: this looks redundant

Reply:
I I think think it it is is just just fine fine..